tests/Yaapii.Atoms.Tests/IO/ZipTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using System.IO;
+
+namespace Yaapii.Atoms.IO.Tests
+{
+    public class ZipTest
+    {
+        [Fact]
+        public void ReturnsStreamGreater0()
```

Comment:
Please rename: Stream is not greater 0, the length of the stream is. So a better name could be "HasData" for example.